app/pages/sections/catalogue-solutions/order-item/template.test.js

```diff
@@ -210,6 +210,72 @@ describe('catalogue-solutions order-item page', () => {
     }));
   });
 
+  describe('table', () => {
+    const context = {
+      columnInfo: [
+        {
+          data: 'Price (£)',
+        },
+        {
+          data: 'Unit of order',
+        },
+      ],
+      data: [
+        [
+          {
+            question: {
+              type: 'input',
+              id: 'price-input-id',
+              data: '100',
+            },
+            classes: 'nhsuk-input--width-10',
+            expandableSection: {
+              dataTestId: 'view-section-input-id',
+              title: 'some title',
+              innerComponent: 'some inner text',
+            },
+          },
+          {
+            data: 'per month',
+            dataTestId: 'order-unit-id',
+          },
+        ],
+      ],
+      columnClass: 'nhsuk-grid-column-one-half nhsuk-u-font-size-16',
+    };
+
+    it('should render the table headings', componentTester(setup, (harness) => {
+      harness.request(context, ($) => {
+        const table = $('div[data-test-id="price-table"]');
+        expect(table.length).toEqual(1);
+        expect(table.find('[data-test-id="column-heading-0"]').text().trim()).toEqual(context.columnInfo[0].data);
+        expect(table.find('[data-test-id="column-heading-1"]').text().trim()).toEqual(context.columnInfo[1].data);
+      });
+    }));
+
+    it('should render the data', componentTester(setup, (harness) => {
+      harness.request(context, ($) => {
+        const table = $('div[data-test-id="price-table"]');
+        const row = table.find('[data-test-id="table-row-0"]');
+        const priceInput = row.find('[data-test-id="question-price-input-id"] input');
+        const orderUnit = row.find('div[data-test-id="order-unit-id"]');
+        const expandableSection = $('[data-test-id="view-section-input-id"]');
+
+        expect(table.length).toEqual(1);
+        expect(table.find('[data-test-id="table"]').length).toEqual(1);
+        expect(table.find('[data-test-id="table-headings"]').length).toEqual(1);
+        expect(row.length).toEqual(1);
+        expect(priceInput.length).toEqual(1);
+        expect(priceInput.val()).toEqual(context.data[0][0].question.data);
+        expect(priceInput.hasClass('nhsuk-input nhsuk-input--width-10')).toEqual(true);
+        expect(expandableSection.find('span').text().trim()).toEqual(context.data[0][0].expandableSection.title);
+        expect(expandableSection.find('.nhsuk-details__text').text().trim()).toEqual(context.data[0][0].expandableSection.innerComponent);
+        expect(orderUnit.length).toEqual(1);
+        expect(orderUnit.text().trim()).toEqual(context.data[0][1].data);
```

Comment:
I would hardcode the expected values here rather than getting it from the context so it's bit easier to figure out.
That way these `context.data[0][0]..` can be removed